tests/dummy/config/targets.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
Is this correct? (it lists IE and Safari)

Reply:
It's just the `dummy` app, so doesn't matter. These are also just the defaults updated from `ember init`.